plugins/inputs/t128_graphql/t128_graphql.go

```diff
@@ -240,10 +225,9 @@ func (plugin *T128GraphQL) createRequest() (*http.Request, error) {
 
 	request.Header.Add("Content-Type", "application/json")
 
-	if plugin.Deadline.Duration != DefaultDeadline {
-		deadline := int(plugin.Deadline.Duration.Truncate(time.Second))
-		request.Header.Add("deadline", fmt.Sprintf("%d", deadline))
-	}
+	// ignored in older versions of 128T
+	deadline := int(plugin.Timeout.Duration.Seconds() - timeoutDeadlineDiff.Seconds())
```

Comment:
If someone put 1s or 0s for timeout? Does graph handle `0` or `-1` deadline?

Reply:
Dillon's going to add a guard to ignore negative and 0sec deadlines. So if a user configures `timeout = "1s"` then `deadline = 0s` which will be treated by web as an infinite deadline. But we're still covered by the 1s timeout